app/controllers/items_controller.rb

```diff
@@ -24,6 +24,7 @@ def update
 
   def destroy
     @todo_to_delete = Item.find(params[:id])
+    @todo_to_delete.subitems.destroy_all
```

Comment:
https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-dependent

Reply:
http://rusrails.ru/active-record-associations#dependent